### DISCUSSION TITLE
get status code and description from call_method

### DISCUSSION THREAD

**Discussion by ronytigo** [FreeOpcUa/opcua-asyncio#1341] [General]
In previous version I used:
`call_method_full(parent_node, node, *arguments)` and I got an object containing the OutputArguments and a StatusCode object with doc, name and value.
Now I call `parent_node.call_method(node, *arguments)` and the result is array of 3 items. The first is the result, and the other are two ints (with value 0 in my case).

**Comment by schroeder-** [comment]
I dont get the problem. The `call_method` will throw a excpetion if the statuscode is bad, otherwise it will return it output arguments. If you want to know which output args are passed back, you can read the value of child "OutputArgument" of the method. 
Or you just call `call_method_full `if you want all values.
